health_check.py

```
@@ -0,0 +1,41 @@
+            ("Ping Server", cog.pingServerLoop),
+            ("Database Backup", cog.backupDatabaseLoop),
+            ("Booster Roles", cog.removeExpiredClaimedBoosterRoles),
+            ("Booster Channels", cog.removeExpiredClaimedBoosterChannels),
+            ("Scheduled Messages", cog.sendScheduledMessages),
+            ("Twitch Polling", cog.pollTwitchStreams),
+        ]
+
+        failed_loops = [name for name, task in loops if not task.is_running()]
+
+        if failed_loops:
+            return HealthCheckResult(self.name, HealthStatus.DEGRADED, "Stopped loops: {', '.join(failed_loops)}")
```

Flake8 (7.3.0)

[error] undefined name 'HealthCheckResult'

(F821)